mne/time_frequency/tests/test_tfr.py

```diff
@@ -264,7 +276,12 @@ def test_tfr_multitaper():
                                             time_bandwidth=4.0, picks=picks)
     power_evoked = tfr_multitaper(epochs.average(), freqs=freqs,
                                   n_cycles=freqs / 2., time_bandwidth=4.0,
-                                  return_itc=False)
+                                  return_itc=False, average=False).average()
```

Comment:
why have `average` param at all if it's so easy just to do `.average()` at the end?